### PR TITLE
RHCEPHQE-16085 -[tools] file based io filler utility

### PR DESCRIPTION
**Pull request by arunravi-ibm**
I/O Filler is a Container based tool to create Background traffic on a file-based share to a container

- It writes random-files of Input size to `/data` 
- It rolls the file over when file-size is reached (default `1G`)
- When a max capacity (default `10G`) is reached, it deletes an input percentage of files (default `70%`)
- And again, starts writing random files to `/data`

This tool keeps on doing above operations till it is externally stopped
Please see - https://issues.redhat.com/browse/RHCEPHQE-16085

### CODE REVIEW

File: roles/iofiller/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# iofiller
+
+This role creates a iofiller image on client nodes that can be used to create multiple contianer to fill 
+a filebased or block device to fill to certain extent in a loop.
+
+The purpise is to fill the target by creating random files of an input size till the storage is filled to a define treshold.
```
Comment (psathyan) [inline roles/iofiller/README.md]: `s/purpise/purpose/g`

File: roles/iofiller/files/Containerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest
+COPY yum.repos.d/* /etc/yum.repos.d/
```
Comment (psathyan) [inline roles/iofiller/files/Containerfile]: Can we remove the repo files post the deployment of required packages?

File: roles/iofiller/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
+
+- name: Copy repo files
+  ansible.builtin.copy:
+    src: "/etc/yum.repos.d"
+    dest: "{{ neerali_iofiller_artifactdir }}/files"
+    remote_src: yes
+    owner: "{{ ansible_user_id }}"
+    group: "{{ ansible_user_gid }}"
```
Comment (psathyan) [inline roles/iofiller/tasks/main.yaml]: Do we need this step?
Can we explore `--volume` during `podman build`?

File: roles/iofiller/README.md (hunk 1 above)
Comment (arunravi-ibm) [reply roles/iofiller/README.md]: Purpose typo is now fixed

File: roles/iofiller/files/Containerfile (hunk 2 above)
Comment (arunravi-ibm) [reply roles/iofiller/files/Containerfile]: With volume mount, we no longer copy yum.repos.d/*

File: roles/iofiller/tasks/main.yaml (hunk 3 above)
Comment (arunravi-ibm) [reply roles/iofiller/tasks/main.yaml]: Yes, I have tested volume and it works!!
removed Copy repo files task

**Review by psathyan — APPROVED** [review]

**Pull request merged**